pkgs/top-level/static.nix

```diff
@@ -117,6 +117,11 @@ in {
     gssSupport = false;
   };
 
+  libconfig = super.libconfig.overrideAttrs (old: {
+    # building the examples fails because they are linked dynamically
+    configureFlags = (old.configureFlags or [ ]) ++ [ "--disable-examples" ];
```

Comment:
Could you move this to the package itself using by checking the `stdenv.hostPlatform.isStatic` flag?

Reply:
Good suggestion, that's much cleaner! Thanks. I have moved it.